Frontend/src/app/app.routes.ts

```diff
@@ -2,9 +2,15 @@ import { Routes } from '@angular/router';
 import { HomeComponent } from './home/home.component';
 import { PageNotFoundComponent } from './page-not-found/page-not-found.component';
 import { RestaurantDetailsComponent } from './restaurant-details/restaurant-details.component';
+import { AuthGuard } from './authentication/auth-guard';
 
 export const routes: Routes = [
-	{ path: '', component: HomeComponent },
+	{
+		path: '',
+		component: HomeComponent,
+		canActivate: [AuthGuard],
+		// data: { roles: ['ADMIN'] },
```

Comment:
I doubt you will want to guard your home page with admin permissions

Reply:
It is just a test